daemon/src/maker_inc_connections.rs

```diff
@@ -41,6 +55,7 @@ pub struct Actor {
     taker_msg_channel: Box<dyn MessageChannel<FromTaker>>,
     noise_priv_key: x25519_dalek::StaticSecret,
     heartbeat_interval: Duration,
+    setup_actors: HashMap<OrderId, xtra::Address<setup_maker::Actor>>,
```

Comment:
I think we may also need a message back to this actor to clean this up, otherwise we have unbounded growth of this hashmap.

Reply:
I copied your solution :D Should we do the same for the CFD actors?

Reply:
I think so yes.

Reply:
I think there might even be an abstraction where the entire `AddressMap` is an actor so this pattern of "cleaning up after the actor is dead" is encapsulated. Something to mull over ...